hack/ocm-minikube-ramen.sh

```diff
@@ -80,6 +80,7 @@ ramen_deploy()
 	spec:
 	  provisioner: rook-ceph.rbd.csi.ceph.com
 	  parameters:
+        schedulingInterval: "1h"
```

Comment:
Keep this at 2m as we want this to work in e2e, and not take an hour to get the data across (I would even suggest 1m)

Reply:
ACK. Making it 1m.